e2e/tests/recordings.spec.ts

```diff
@@ -10,7 +12,7 @@ test.beforeEach(async ({page, context}) => {
 });
 
 test.describe('call recordings', () => {
-    test.use({storageState: userState.users[6].storageStatePath});
```

Comment:
This was the line causing the flakyness. Forgot to update the index which meant it could occasionally conflict with other tests running in parellel.